clients/js/test/createV1.test.ts

```diff
@@ -258,6 +258,32 @@ test('it can create a collection NonFungible', async (t) => {
   });
 });
 
+test('it can create a collectionV2 NonFungible', async (t) => {
+  // Given a new mint Signer.
+  const umi = await createUmi();
+  const mint = generateSigner(umi);
+
+  const collectionV2Padding = new Array(8).fill(0);
+
+  // When we create a new NonFungible at this address.
+  await createV1(umi, {
+    mint,
+    name: 'My Collection NFT',
+    uri: 'https://example.com/my-collection-nft.json',
+    sellerFeeBasisPoints: percentAmount(5.5),
+    isCollection: true,
+    collectionDetails: collectionDetails('V2', { padding: collectionV2Padding }),
```

Comment:
Now that I see we default to V1 in js client, I think we should probably update the resolver to default to V2:
https://github.com/metaplex-foundation/mpl-token-metadata/blob/main/clients/js/src/hooked/resolvers.ts#L27-L33

wdyt @blockiosaurus ?

Then the two tests here will need to be switched as well.

Also, I do not think we have any default in the Rust client, but wondering if we should add that.